paket.lock

```diff
@@ -878,4 +878,4 @@ GROUP Build
 NUGET
   remote: https://api.nuget.org/v3/index.json
     FAKE (5.0.0-beta010)
-    GitVersion.CommandLine (3.6.5)
+    GitVersion.CommandLine (4.0.0-beta0012)
```

Comment:
Is there something specific we need from the 4.0-pre chain? Why not stick with the stable release?

Reply:
there is GitTools/GitVersion#1043 which suggested the Gitversion error was already fixed. So I tried that. but then I had different error